doc/Tutorials.xml

```diff
@@ -181,12 +181,108 @@ There are various examples of packages using &AutoDoc; for only
 this purpose, e.g. <Package>IO</Package> and <Package>orb</Package>.
 <P/>
 
-<!--
-TODO: Explain how to mix AutoDoc content with existing &GAPDoc; XML files.
--->
+<Subsection Label="Tut:IntegrateExisting:EverythingThere">
+  <Heading>Using &AutoDoc; on a complete &GAPDoc; manual</Heading>
+
+Suppose you already have a complete XML manual, with some main and title
+XML files and some documentation for operations distributed over
+all your <F>.g{,d,i}</F> files. Even then &AutoDoc; can do something for you.
+Suppose your main xml file is named <F>PACKAGENAME.xml</F> and is in the
+<F>/doc</F> subfolder of your package. Then calling
+<Listing>
+LoadPackage( "AutoDoc" );
+AutoDoc( : autodoc := false );
+</Listing>
+will rebuild your manual. The main point in favor of doing is that you do
+not have to keep a list of all the <F>.g{,d,i}</F> files you have manual entries in.
```

Comment:
Actually, I think there is a bit more to that... I was also working on some text for this, and this is what I wrote; perhaps you can salvage some of it? Note that I started from a slightly different angle (i.e. the assumption that there is an existing `makedocrel.g` which can be used as a starting point). Of course your approach is fine, too!
```
In order to build a &GAPDoc; based manual, one typically uses a
<F>makedoc.g</F> or <F>makedocrel.g</F> file which might
look something like this:

@BeginCode c1
LoadPackage( "GAPDoc" );
MakeGAPDocDoc( "doc", "manual", [ "../PackageInfo.g" ],
               "FooBar", "../../..", "MathJax");; 
CopyHTMLStyleFiles( "doc" );
GAPDocManualLab( "FooBar" );; 
@EndCode
@InsertCode c1

With &AutoDoc;, a roughly equivalent <F>makedoc.g</F> might look like this:

@BeginLog
if fail = LoadPackage("AutoDoc", ">= 2016.01.21") then
    Error("AutoDoc 2016.01.21 or newer is required");
fi;
AutoDoc(rec(gapdoc:=rec(main:="manual")));
@EndLog
```